tests/model_serving/model_server/components/kserve_dsc_deployment_mode/conftest.py

```diff
@@ -54,6 +55,54 @@ def patched_default_deployment_mode_in_dsc(
     )
 
 
+@pytest.fixture(scope="class")
+def default_deployment_serviceconfig_in_dsc(
```

Comment:
why is this fixture needed? doesn;t it do what `patched_default_deployment_serviceconfig_in_dsc` does?

Reply:
id did some change please let me know if its fine